common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -989,3 +989,52 @@ sBssidInfo:
   downlink_estimated_mac_data_rate_mbps: uint32_t
   uplink_estimated_mac_data_rate_mbps: uint32_t
   sta_measured_uplink_rssi_dbm_enc: uint8_t
+
+sClientConfig:
+  _type: struct
+  stay_on_initial_radio:
+    _type: uint8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  stay_on_selected_device:
+    _type: uint8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  selected_bands:
+    _type: char
+    _length: ["beerocks::message::SELECTED_BANDS_LENGTH"]
+  time_life_delay_days:
+    _type: uint32_t
+    _comment: optional parameter, Timelife delay of the client configuration (if configured), otherwise 0.
+
+sClient:
+  _type: struct
+  sta_mac:
+    _type: sMacAddr
+    _comment: Client MAC
+  timestamp_sec:
+    _type: uint32_t
+    _comment: Time of last client update (in Seconds)
+  stay_on_initial_radio:
+    _type: uint8_t
```

Comment:
replace `uint8_t` with `int8_t` since this parameter can receive `-1`
(this comment is related to all tlvf parameters that support `-1` as "not-configured" value

Reply:
You're right, my mistake, changed